osutil/lookup_path_test.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil_test
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/osutil"
+
+	. "gopkg.in/check.v1"
+)
+
+type lookupPathSuite struct {
+	basePath string
+}
+
+var _ = Suite(&lookupPathSuite{})
+
+func (s *lookupPathSuite) TestGivesCorrectPath(c *C) {
```

Comment:
Similarly, let's integrate these into stat_test.go.

For the record, this package is overbroken, so I apologize for the bad example. We should eventually merge some of those tiny one-function files into proper chunks by similarity. A bit like stat.go.